### ISSUE TITLE
Process all years

### ISSUE THREAD

**Issue by matthewkmayer** [matthewkmayer/rusty-von-humboldt#19]
Right now our request to S3 is limited to a single year because we specify the key must start with the year.

Support *not* providing the year argument and process everything. 🤘 

Keep the year arg as optional.

**Comment by matthewkmayer** [comment]
Limiting to a single year is actually a bit useful since we can run multiple processes in parallel by running another copy of RvH. Closing, won't do for the moment.
